laravel-app/app/Http/Controllers/PostController.php

```diff
@@ -2,26 +2,38 @@
 
 namespace App\Http\Controllers;
 
-use Illuminate\Http\Request;
 use App\Http\Requests\UserPostRequest;
-use App\Models\UsersPost;
+use Illuminate\View\View;
 
 class PostController extends Controller
 {
+    /**
+     * Show login form page.
+     *
+     * @return View
+     */
```

Comment:
Since they are using Laravel 10. Laravel already removed doc block and used return type declaration.

Docs:
https://laravel.com/docs/10.x/releases#types
https://www.php.net/manual/en/language.types.declarations.php

Demo:
https://www.reddit.com/r/laravel/comments/11328e0/quick_demo_of_the_laravel_10x_shift/